lms/djangoapps/commerce/models.py

```diff
@@ -35,9 +36,16 @@ class Meta(object):
     )
     receipt_page = models.CharField(
         max_length=255,
-        default='/commerce/checkout/receipt/?orderNum=',
+        default='/checkout/receipt/?orderNum=',
         help_text=_('Path to order receipt page.')
     )
+    site = models.ForeignKey(
```

Comment:
We'll need this ForeignKey so that we may define switch back and forth between the two sites (LMS and Otto). I can, however, move the receipt page there so that we may define the relative path for each site.